support/cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/authenticator/OAuth20ClientIdClientSecretAuthenticatorTests.java

```diff
@@ -28,24 +40,16 @@
 @Tag("OAuth")
 @DirtiesContext
 public class OAuth20ClientIdClientSecretAuthenticatorTests extends BaseOAuth20AuthenticatorTests {
-    protected OAuth20ClientIdClientSecretAuthenticator authenticator;
-
-    @BeforeEach
-    public void init() {
-        authenticator = new OAuth20ClientIdClientSecretAuthenticator(servicesManager,
-            serviceFactory,
-            new RegisteredServiceAccessStrategyAuditableEnforcer(),
-            new OAuth20RegisteredServiceCipherExecutor(),
-            ticketRegistry,
-            defaultPrincipalResolver);
-    }
+
+    @Autowired
+    private Authenticator oAuthClientAuthenticator;
 
     @RetryingTest(3)
     public void verifyAuthentication() {
```

Comment:
All such cases needs to be moved to an inner class, per the previous suggestion.

Reply:
Ok, I thought it was enough to only have an inner class for the special config. Done.